package Exceptions;

public class UsuarioNaoEncontradoException extends RuntimeException {
    public String getMessage() {
        return "Usuario n√£o encontrado";
    }
}